source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -154,6 +154,23 @@ void BioXASAppController::onRegionOfInterestBoundingRangeChanged(AMRegionOfInter
 		xasConfiguration_->setRegionOfInterestBoundingRange(region);
 }
 
+void BioXASAppController::goToBeamStatusView(AMControl *control)
```

Comment:
That name does describe part of what happens in the `goToBeamStatusView(AMControl*)`, the part where the selected component of the beam status view is set. But it doesn't describe the part where the current main window pane is set to the beam status view.